src/transformers/models/qdqbert/tokenization_qdqbert_fast.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
```

Comment:
Same this file can be removed with the comment above.

Reply:
addressed in commit: https://github.com/huggingface/transformers/pull/14066/commits/873d35220d770907d4af417bdd260071e35a5e28